contracts/strategies/convex/ConvexStrategyEURTMainnet.sol

```diff
@@ -16,19 +16,18 @@ contract ConvexStrategyEURTMainnet is ConvexStrategyUL {
     address rewardPool = address(0xD814BFC091111E1417a669672144aFFAA081c3CE);
     address crv = address(0xD533a949740bb3306d119CC777fa900bA034cd52);
     address cvx = address(0x4e3FBD56CD56c3e72c1403e103b45Db9da5B9D2B);
-    address eurt = address(0xC581b735A1688071A1746c968e0798D642EDE491);
+    address dai = address(0x6B175474E89094C44Da98b954EedeAC495271d0F);
```

Comment:
Think something might have got switched up here. This pool does not have DAI in it. Only EURT and sEUR. This is an existing strategy that you seem to have altered.